tests/testcases/mod.rs

```diff
@@ -321,7 +327,23 @@ pub fn bootstrap_smoke_test(
     };
     let mut build = std::process::Command::new(spin_binary());
     // Ensure `spin` is on the path
-    build.env("PATH", path).args(["build"]);
+    build.env("PATH", &path).args(["build"]);
+    build_env_vars.iter().for_each(|(key, value)| {
+        if key == "PATH" {
+            let mut custom_path = value.to_owned();
+            if value.starts_with('.') {
+                let current_dir = env.path();
+                custom_path = current_dir
+                    .join(value)
+                    .to_str()
+                    .unwrap_or_default()
+                    .to_owned();
+            }
+            build.env(key, format!("{}:{}", custom_path, path));
```

Comment:
This works today because `build.env("PATH")` has not been modified since it was set to `path`, but feels a bit fragile.  If you want to prepend to the command PATH, it would be good to use the command PATH as the source of truth.  (I don't know if this will ever be an issue in practice though!)